src/DynamoCore/Search/SearchElements/NodeModelSearchElement.cs

```diff
@@ -12,19 +12,25 @@ namespace Dynamo.Search.SearchElements
     public class NodeModelSearchElement : NodeModelSearchElementBase
     {
         private readonly Func<NodeModel> constructor;
+        private readonly string creationName;
 
+        public override string CreationName { get {return creationName;} }
+        
```

Comment:
@mjkkirschner I checked the concrete type for a UI node like `ExportToSAT` looks like `"ConcreteType": "GeometryUI.ExportWithUnits, GeometryUI"` in the JSON file, while the `CreationName` in this case after this change will just be `GeometryUI.ExportWithUnits`, the type name of the UI node class. While the `CreationName` for zero touch nodes, for example, will continue to be the fully qualified method name: "Autodesk.DesignScript.Geometry.Geometry.ExportToSAT".